tsl/src/nodes/decompress_chunk/exec.c

```diff
@@ -212,7 +212,7 @@ decompress_chunk_begin(CustomScanState *node, EState *estate, int eflags)
 										node->ss.ss_ScanTupleSlot->tts_tupleDescriptor);
 		}
 	}
-	/* Sort keys should only be present when sorted_merge_append is used */
+	/* Sort keys should only be present when batch sorted merge is used. */
```

Comment:
Even without batch sortetd merge we might still want to push down ordering and skip the ordering after decompression.

Reply:
Yes, we do that, but these are the keys for the sorting performed inside the DecompressChunk node itself. The only case when it does that is for batch sorted merge, otherwise the sorting is performed by the underlying compressed scan.